### ISSUE TITLE
Incorrect Letter Case Name

### ISSUE THREAD

**Issue by SavPhill** [WordPress/gutenberg#48617]
## What problem does this address?

In the Settings Sidebar, the Typography Panel incorrectly labels the letter case where every first word in the sentance is a capital letter, as 'Capitalize'. 

![image](https://user-images.githubusercontent.com/38789408/222091912-036a988b-fccc-4975-a5dd-bb0a47817af4.png)

The correct terminology is either 'Initial Caps' or 'Start Case', as Capitalize is not an official case variation, and doesn't describe the behavior of the output.

## What is your proposed solution?
Rename the label to 'Initial Caps' which I have addressed in the following PR: 

https://github.com/WordPress/gutenberg/pull/48613

**Comment by aristath** [comment]
This is the name of the CSS property: https://developer.mozilla.org/en-US/docs/Web/CSS/text-transform
`capitalize` makes the 1st letter of the word uppercase.

**Comment by richtabor** [comment]
> This is the name of the CSS property:

I agree. It's a bit of intersection between type and CSS (could even be "Title casing"). 

I do think however that "Capitalize" is the sensible approach for typography settings here though.
